resources/views/components/listing.blade.php

```diff
@@ -26,7 +26,7 @@
     >
         <div>
             <ais-instant-search
-                v-if="loaded"
+                v-if="loaded && searchClient"
```

Comment:
Why isn't `loaded` enough? Multiple options will be forgotten easily.

Reply:
loaded can be set to true before searchClient is available, and it is absolutely required to start mounting `ais-instant-search`

Reply:
Can we couple those so only one condition is needed within a template?